### PR TITLE
Polygon_mesh_processing:  Add performance comparison for hole filling in User Manual

### PR DESCRIPTION
**Pull request by afabri**
## Summary of Changes

Add a benchmark for hole filling  (with/without Delaunay).

## Release Management

* Affected package(s): Polygon_mesh_processing (only doc)

### CODE REVIEW

File: Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt [hunk 1]
```diff
@@ -342,6 +342,27 @@ iteratively filled, refined and faired to get a faired mesh with no hole.
  Holes in the fork model are filled with triangle patches.
  \cgalFigureEnd
 
+subsection HFPerformance Performance
```
Comment (sloriot) [inline Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt]: missing backslash

File: Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt [hunk 2]
```diff
@@ -342,6 +342,27 @@ iteratively filled, refined and faired to get a faired mesh with no hole.
...
+
+The hole filling algorithm has a complexity which depends on the
+number of vertices.  While \cgalCite{liepa2003filling} has a running
+time of \f$ O(n^3)\f$ , YY in most cases has running time of \f$ O(n
```
Comment (sloriot) [inline Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt]: YY?

File: Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt [hunk 3]
```diff
@@ -342,6 +342,27 @@ iteratively filled, refined and faired to get a faired mesh with no hole.
...
+\log n)\f$.  We were running `triangulate_refine_and_fair_hole()` for
+the below meshes (and two more meshes with smaller holes.
```
Comment (sloriot) [inline Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt]: incomplete sentence and missing ).

File: Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt [hunk 4]
```diff
@@ -342,6 +342,27 @@ iteratively filled, refined and faired to get a faired mesh with no hole.
  Holes in the fork model are filled with triangle patches.
  \cgalFigureEnd
 
+subsection HFPerformance Performance
+
+The hole filling algorithm has a complexity which depends on the
+number of vertices.  While \cgalCite{liepa2003filling} has a running
+time of \f$ O(n^3)\f$ , YY in most cases has running time of \f$ O(n
+\log n)\f$.  We were running `triangulate_refine_and_fair_hole()` for
```
Comment (sloriot) [inline Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt]: You should give some idea on the config + the release of CGAL for which it was mesured.

**Comment by lrineau** [comment]
This PR fails in Travis because of a bad permission on a file:
```
Polygon_mesh_processing/doc/Polygon_mesh_processing/fig/elephants-with-holes.png
```
https://travis-ci.org/CGAL/cgal/jobs/245370286#L4346

**Comment by lrineau** [comment]
Documentation error:
```
/home/cgal-testsuite/cgal_doc_build/CGAL-4.11-Ic-52/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt:388: Warning: image file elephants-with-hole.png is not found in IMAGE_PATH: assuming external image.
/home/cgal-testsuite/cgal_doc_build/CGAL-4.11-Ic-52/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt:389: Warning: image file elephants-with-hole.png is not found in IMAGE_PATH: assuming external image.
```
https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-4.11-Ic-52/Polygon_mesh_processing.log

**Pull request merged**